CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Bug fixes
 
+* [#6821](https://github.com/rubocop-hq/rubocop/pull/6821): Fix false positive for Rails/LinkToBlank when _blank is a symbol. ([@Intrepidd][])
```

Comment:
What is not able to find a problem is "false negative" (not "false positive") :-)

Reply:
Ok I thought as a false positive as in "your code is ok" when it is not.

I'll update my PR right away

Thanks !